RetailCoder.VBE/UI/Command/MenuItems/ParentMenus/ParentMenuItemBase.cs

```diff
@@ -92,7 +92,7 @@ public void RemoveMenu()
             Logger.Debug($"Removing menu {_key}.");
             RemoveChildren();
             Item?.Delete();
-            Item?.Release();
+            Item?.Dispose();
             Item = null;
```

Comment:
Are we missing a `Parent.Dispose()` here? The class seems to have a [`Parent`](https://github.com/rubberduck-vba/Rubberduck/pull/3635/files#diff-0988dc676c4156b23d3ac442937e78b2R26)

Reply:
Except for the top-level menu items, the `Parent` is another `ParentMenuItem`, possibly with more children. We dispose of Thom in their own removal code. The `AppMenu` deals with the parents of the top-level menus.